src/modules/theme/constants/colors.js

```diff
@@ -19,7 +21,9 @@ export const brightColors: ColorsType = {
   textSecondaryColor: '#585858',
   textDecorationColor: '#c7c7c7',
   textDisabledColor: '#d0d0d0',
-  backgroundColor: '#ffffff'
+  backgroundColor: '#ffffff',
+  tunewsThemeColor: '#007aa8',
+  tunewsThemeColorLight: 'rgba(0, 122, 168, 0.4)'
```

Comment:
If possible this should be hex.

Reply:
ASFAIK rgba is more supported than 8 character hex notation so I prefer to use it.
It's an easy switch if you think browser support won't be an issue

Reply:
No, that's fine.